vmdb/lib/workers/event_catcher_kubernetes.rb

```diff
@@ -5,7 +5,7 @@ def event_monitor_handle
     require 'kubernetes/events/kubernetes_event_monitor'
     @event_monitor_handle ||= KubernetesEventMonitor.new(
       @ems.api_endpoint,
-      @ems.api_version
+      EmsKubernetes.kubernetes_version
```

Comment:
> Does it ever make sense that we would have different versions for different instances of EmsKubernetes? I'm wondering if there should be an instance method that just delegates to the class method. Then, in the future, you could potentially have that method become a column or something else and wouldn't have to change things all over.

@Fryguy what you're proposing is looping back on the same error we had to begin with. API version to use is a runtime decision based on what the MIQ code is able to handle, not a property of the provider instance.

Selection of the proper/best api version to use is made at handshake with the endpoint (in this case we handle just `v1beta3` / `v1`.)